### ISSUE TITLE
[docs] do not suggest install with `pip install jpterm`

### ISSUE THREAD

**Issue by baggiponte** [davidbrochart/jpterm#81]
Thanks for making jpterm!

I would like to suggest to avoid suggesting to install stuff with `pip install`. For CLI-based tools, pythonistas should use [`pipx`](https://pipx.pypa.io/stable/). Unfortunately, `pip install` and even `pip install --user` can mess with your system python.

I could open a PR highlighting the best practices and suggest a recommended workflow, and/or suggest other tools that enable "global installs" like `pixi` (AFAIK only rye right now).

**Comment by davidbrochart** [comment]
I'm not familiar with `pipx`, but I've seen e.g. `toolong` [suggests to use it](https://github.com/Textualize/toolong?tab=readme-ov-file#how).
The way `jpterm` is currently released is by [exactly pinning dependencies](https://github.com/davidbrochart/jpterm/blob/ea3be08c887c50acf7568b17245d74846174fe45/pyproject.toml#L28-L47), so I don't know if `pipx` will bring something better?
Happy to be proved wrong, if you want to open a PR :smile:

**Comment by baggiponte** [comment]
Hey there, I was working on a PR and I realised one thing. If jpterm works like jupyter, you might want to have it installed in a venv or use ipykernel, am I right?

I set out to test that and I noticed the CTRL + E shortcut does not run any cell. I am on the latest macOS and I tried both with alacritty and macOS Terminal app.

**Comment by davidbrochart** [comment]
Yes, `ipykernel` should be [installed](https://github.com/davidbrochart/jpterm/blob/bd35c2dd90cff9e378972659ca40a47d07f69600/plugins/local_kernels/pyproject.toml#L28). Maybe Ctrl-E is caught by your shell, doesn Ctrl-R work?

**Comment by baggiponte** [comment]
> Yes, `ipykernel` should be [installed](https://github.com/davidbrochart/jpterm/blob/bd35c2dd90cff9e378972659ca40a47d07f69600/plugins/local_kernels/pyproject.toml#L28). Maybe Ctrl-E is caught by your shell, doesn Ctrl-R work?

Oh definitely, you are right. They are both caught by the shell (ctrl-e is inverse history search; ctrl-e is "edit current command in $EDITOR". Can I change them?

**Comment by davidbrochart** [comment]
On my terminal Ctrl-E goes at the end of the line, and Ctrl-R is `reverse-i-search`, but it works in Textual anyway. I'm wondering why Textual doesn't overwrite these key bindings for you, maybe worth opening an issue in Textual? It's not currently possible to change key bindings in `jpterm`.
